hiss/team/tests/view_tests/detail.py

```diff
@@ -25,7 +25,7 @@ def test_fails_if_user_not_applied(self):
 
         response = self.client.get(reverse_lazy("team:detail", args=[team.pk]))
 
-        self.assertEqual(response.status_code, 403)
+        self.assertRedirects(response, "/status/")
```

Comment:
Please use a `reverse_lazy` here

```suggestion
        self.assertRedirects(response, reverse_lazy("status"))
```